src/ontology/cl.Makefile

```diff
@@ -20,6 +20,11 @@
 #tmp/pr_logical.owl: mirror/pr.owl
 #	echo "Skipped pr logical" && cp $< $@
 
+.PHONY: rdiff
+rdiff: 
+	curl https://raw.githubusercontent.com/shawntanzk/cell-ontology/master/src/ontology/cl-edit.owl -o rdifftmp.owl
+	$(ROBOT) diff -labels True -left rdifftmp.owl --right cl-edit.owl -format markdown --output diff.md
```

Comment:
For now this is fine @ubyndr it would be important that however you realised the final diff system we integrate in ODK, that it can be run on _any_ file in the ODK pipeline.

Reply:
Can we make this dependent on file extension?

Reply:
exactly.. I agree. It should be something like:

```
rdiff-%: tmp/main-% tmp/branch-%*
	ROBOT diff tmp/main-$* tmp/branch-$* ....
```

Reply:
and if it is, say cl.owl, it should depend on cl.owl so that it is actually created by make first!

Reply:
does this mean we want to compare $(SRC).owl that is made from $(SRC)-edit.owl?